packages/components/src/tabs/test/index.tsx

```diff
@@ -1260,27 +1026,7 @@ describe( 'Tabs', () => {
 		} );
 		describe( 'Without `initialTabId` prop', () => {
 			it( 'should render the tab specified by the `specifiedTabId` prop', async () => {
```

Comment:
```suggestion
			it( 'should render the tab specified by the `selectedTabId` prop', async () => {
```

Let's make sure that `specifiedTabId` is not used anywhere else

Reply:
Looks like it was just that once, thank you!